adafruit_rfm69.py

```diff
@@ -358,7 +358,26 @@ def __init__(
         self.encryption_key = encryption_key
         # Set transmit power to 13 dBm, a safe value any module supports.
         self.tx_power = 13
-        # last RSSI reading
+        # initialize timeouts and delays delays
+        self.ack_wait = 0.5
+        self.receive_timeout = 0.5
+        self.xmit_timeout = 2.0
+        self.ack_retries = 5
```

Comment:
Do you want folks to be able to change these directly? If so, to get them in documentation they'll need `"""<doc string>"""` docs after them.

Reply:
Yes - will add docstrings.

Reply:
done